custom_widgets/annotation_manager/annotationManager.py

```diff
@@ -73,99 +87,243 @@ def generateAnnotationItem(self, annotation: Any) -> None:
         """ Generates an annotation item """
         # Finding the annotation's class
         if annotation.className not in self._classes:
-            # An annotation is dependant on its class item being present
+            # An annotation is dependent on its class item being present
             _class = self.getClass(annotation.className)
             self.generateClassItem(_class.className, _class.classColour)
 
-        self.addAnnotationToClassItem(annotation.className, annotation.id)
-
-    def addClassSelectionListItem(self, className: str, classColour: tuple) -> None:
-        """ Adds a class item to the class selection list widget """
-        classListItem = ClassItem(className, classColour, self.themePaletteColours, self.fontRegular, self.fontTitle, app=self)
-        classListItem.annotation_selected.connect(lambda annotation_id: self.annotation_selected.emit(annotation_id))
-        classListItem.annotation_removed.connect(lambda annotation_id: self.annotation_removed.emit(annotation_id))
-        classListItem.annotation_hidden.connect(lambda annotation_id, hidden: self.annotation_hidden.emit(annotation_id, hidden))
+        self.addAnnotationToClass(annotation.className, annotation.id)
 
-        self.classListItems.append(classListItem)
-        self.classSelectionListWidget.addItemToListWidget(classListItem)
+    def addAnnotationToClass(self, className: str, annotationName: str) -> None:
```

Comment:
```suggestion
    def addAnnotationItem(self, className: str, annotationId: str) -> None:
```
Same here. As well, there is no annotation name - only an id